src/desfire/devices/pn532.py

```diff
@@ -11,6 +11,9 @@
     _has_serial = False
 else:
     _has_serial = True
+import logging
```

Comment:
Imports should follow import sorting rules, you can use the provided tasks command or ruff to auto-format that for you